versioned/persist/adapter/build.gradle.kts

```diff
@@ -30,7 +30,9 @@ dependencies {
   compileOnly(platform(project(":nessie-deps-build-only")))
   annotationProcessor(platform(project(":nessie-deps-build-only")))
 
+  implementation(project(":nessie-model"))
   implementation(project(":nessie-versioned-spi"))
+  compileOnly("org.eclipse.microprofile.openapi:microprofile-openapi-api")
```

Comment:
Why? I might have missed the changes that need the new dependencies... :thinking:

Reply:
Annotations leaking into the compilation (otherwise there are compilation warnings)

Reply:
Also: `Content` leaks here as well.